lib/plugins/packages.js

```diff
@@ -87,6 +87,7 @@ async function refreshPackageInfo (packageName, version) {
   const installedPackageVersion = packageJson && projectPackage.dependencies[packageName]
   const installed = !!installedPackageVersion
   const installedLocally = installedPackageVersion?.startsWith('file:')
+  const installedFromGithub = installedPackageVersion?.startsWith('github')
```

Comment:
Probably want to run `resolved` through [`new URL()`](https://nodejs.org/api/url.html#new-urlinput-base) confirming hostname as `registry.npmjs.org`?

```json
{
  "name": "test-prototype",
  "lockfileVersion": 3,
  "packages": {
    "node_modules/govuk-prototype-kit": {
      "version": "13.13.4",
      "resolved": "https://registry.npmjs.org/govuk-prototype-kit/-/govuk-prototype-kit-13.13.4.tgz",
      "integrity": "sha512-P4HfYXIAUpjWe5NIfTEYxOkg0iE9GuQBmJ0KO+KoFVkFre2yPlAoa3kQA30UjjvtT2+c+Sc70Abkx424rwaiAg==",
      "dependencies": {}
    }
  }
}
```

Reply:
For now though, shall we prefer `github:` (with a colon) consistent with `file:`?

```suggestion
  const installedFromGithub = installedPackageVersion?.startsWith('github:')
```

Reply:
Done as suggested